PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +1,24 @@
-## JIRA Link
+### JIRA Link
 
+### Summary
+Please include a summary of the change and add relevant motivation and context. Keep in mind that this information might be used by on-call engineers to handle potential incidents.
 
-## Summary of Change
+### Risks
 
+- Potentially, what could go wrong with this change in the worst case scenario (i.e. not being able to process write requests, etc.)?
+- Note that if there is no risk in applying this change, there is also no risk in **not** doing this change
```

Comment:
can you elaborate on your goal with this sentence? curious if we can have better wording

Reply:
This is my way of begging people to fill this in and not just put "No risk". Reworded it to be more explicit